lib/npm/install.js

```diff
@@ -48,43 +48,36 @@ function install(context, next) {
     return next(Error('Install Timed Out'));
   }
 
-  let installOutput = '';
-  let installError = '';
-
   proc.stderr.on('data', function (chunk) {
+    context.testError.append(chunk);
```

Comment:
It looks like moving the appending of the chunk to before this if-block means you lose ANSI-stripping and newline normalizing in `payload.testOutput`. Was that intentional?